zeppelin-interpreter/src/test/java/org/apache/zeppelin/iotdb/IoTDBInterpreterTest.java

```diff
@@ -389,4 +393,11 @@ public void testListUser() {
     Assert.assertEquals(Code.SUCCESS, actual.code());
     Assert.assertEquals(gt, actual.message().get(0).getData());
   }
+
+  private void assertEqualsNonStrict(String gt, String actual) {
+    List<String> actualList = new ArrayList<String>(Arrays.asList(actual.split("\n")));
+    List<String> expectedList = new ArrayList<String>(Arrays.asList(gt.split("\n")));
+    Assert.assertEquals(expectedList.get(0), actualList.get(0));
+    Assert.assertEquals(new HashSet<String>(expectedList), new HashSet<String>(actualList));
```

Comment:
This modification has a benifit that we can know which line of results are incorrect.
However, IMO, it is sensitive about the order of the resultset table. 
So, using Hashset to compare the result is not very good. 
We may need to use ArrayList to split the result and compare them line by line.